xapian-letor/xapianletor-config.in

```diff
@@ -313,7 +313,7 @@ for arg in $actions ; do
 	fi
 
 	set_L_to_library_path
-	echo "$F$L -lxapian@LIBRARY_VERSION_SUFFIX@ -lxapianletor"
+	echo "$F$L -lxapian -lxapianletor"
```

Comment:
We shouldn't need to explicitly link with `-lxapian`.

And doing so will cause problems if the xapian-core that we're using is suffixed, because we actually want `libxapian-1.5` (say) which will get automatically linked in, but then this will cause us to also link to `libxapian` (which will probably be the system install of 1.4.x).  So we end up linking to two different shared libraries which will provide a lot of the same symbols, and things will probably go wrong in a rather messy and confusing way.

Reply:
fixed in a fixup commit.